jaas/src/main/java/com/griddynamics/jagger/jaas/controller/JaaSRestController.java

```diff
@@ -0,0 +1,78 @@
+ *         created 9/14/16
+ */
+interface JaaSRestController {
+
+    @ApiImplicitParams(
+            @ApiImplicitParam(name = "sessionId", value = "Input session id", required = true, paramType = "path",
+                    dataType = "string")
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Success", response = SessionEntity.class),
+            @ApiResponse(code = 404, message = "Not found"),
+            @ApiResponse(code = 500, message = "Failure")
+    })
+    SessionEntity getSession(String sessionId);
+
+    @ApiImplicitParams(
+            @ApiImplicitParam(name = "id", value = "Input session ids", allowMultiple = true, dataType = "string")
+    )
+    Set<SessionEntity> getSessions(String[] sessionIds);
+
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "sessionId", value = "Input session id", required = true, paramType = "path",
+                    dataType = "string"),
+            @ApiImplicitParam(name = "testName", value = "Input test name", required = true, paramType = "path",
+                    dataType = "string")
+    })
+    TestEntity getTest(String sessionId, String testName);
+
+    @ApiImplicitParams(
+            @ApiImplicitParam(name = "sessionId", value = "Input session id", required = true, paramType = "path",
+                    dataType = "string")
+    )
+    Set<TestEntity> getTests(String sessionId);
+
+    @ApiImplicitParams(
+            @ApiImplicitParam(name = "testId", value = "Input test id", required = true, dataType = "long",
+                    paramType = "path")
+    )
+    Set<MetricEntity> getMetrics(Long testId);
+
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "sessionId", value = "Input session id", required = true, paramType = "path",
+                    dataType = "string"),
+            @ApiImplicitParam(name = "testName", value = "Input test name", required = true, paramType = "path",
+                    dataType = "string")
+    })
+    Set<MetricEntity> getMetrics(String sessionId, String testName);
+
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "sessionId", value = "Input session id", required = true, paramType = "path",
+                    dataType = "string"),
+            @ApiImplicitParam(name = "testName", value = "Input test name", required = true, paramType = "path",
+                    dataType = "string")
+    })
+    Map<MetricEntity, MetricSummaryValueEntity> getMetricsSummary(String sessionId, String testName);
+
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "sessionId", value = "Input session id", required = true, paramType = "path",
+                    dataType = "string"),
+            @ApiImplicitParam(name = "testName", value = "Input test name", required = true, paramType = "path",
+                    dataType = "string")
+    })
+    Map<MetricEntity, List<MetricPlotPointEntity>> getMetricPlotData(String sessionId, String testName);
+}
```

Comment:
where the generated docu will be available. Please, update https://github.com/griddynamics/jagger/blob/master/doc/pages/JaasMain.txt with the link to API docu